ferrocene/doc/internal-procedures/src/manual-upstream-pulls.rst

```diff
@@ -10,97 +10,70 @@ What is an Upstream Pull
 An "upstream pull" is a pull request that pulls in commits from the upstream Rust
 repository back into the Ferrocene repository to keep them in sync. This
 procedure is usually performed by a periodic Github Action that opens a pull
-request automatically, only requiring sign off of a reviewer to get merged.
-Sometimes merge conflicts may occur in which case automation opens an issue
-describing the conflict and requiring the upstream pull to be done manually.
+request automatically. Sometimes merge conflicts may occur in which case the conflict
+needs to to be resolved manually.
 
-Manual intervention when the automation fails
----------------------------------------------
+Upstram Pull without merge conflicts
+---------------------
 
-Performing a manual pull differs slightly depending on whether the pull is done
-for the `master` or `beta` branch. The following text uses a few angle
-bracketed terms that have to be substituted as follows:
+To sign off a automatic upstream pull without merge conflicts, the reviewer only
+needs to approve the PR and comment `bors merge`. This will trigger the "full" CI
+workflow. If the workflow succeeds, the Upstream Pull is getting merged.
 
-* `<BASE_BRANCH>`: The branch of the Ferrocene repository to pull the changes into.
-  This is `main` if doing a `master` upstream pull and `release/1.70` for a
-  `beta` upstream pull.
-* `<UPSTREAM_BRANCH>`: The upstream branch to pull changes from. This is `master`
-  if doing a `master` upstream pull and `beta` for a `beta` upstream pull.
-* `<NEW_PULL_BRANCH>`: This represents the branch you will be making the pull
-  with. Replace it with a branch name of your choice, e.g.
-  `pull-upstream-master-2023-05-22`
+Upstream Pull with merge conflicts
+----------------------------------
 
-Initiating the pull
-^^^^^^^^^^^^^^^^^^^
+If an automatic upstream pull creates merge conflicts, the automation commits the
+merge conflicts and they need to be resolved manually.
 
-Run the following git commands to set up a new local `<NEW_PULL_BRANCH>`
-branch to do the pull on::
+Checkout PR
+^^^^^^^^^^^
 
-  git checkout <BASE_BRANCH> # check out the ferrocene branch to do the pull for
-  git pull --rebase # make sure your local branch is up to date
-  git submodule update # update any submodules that might've changed
-  git checkout -b <NEW_PULL_BRANCH> # checkout a new branch to do the pull with
+Start by checking out the PR. The name of the branch is
+`automation/pull-upstream/<ID>`, e.g. `automation/pull-upstream/dfvjj2s4`::
 
-Performing the pull
-^^^^^^^^^^^^^^^^^^^
+    git switch automation/pull-upstream/<ID>
```

Comment:
If we want to include the git commands someone should use (not sure if we need to), we should also mentioning running `git fetch` first, otherwise the command will fail if the branch didn't exist during your last fetch.